application/models/Logbook_model.php

```diff
@@ -1958,8 +1958,10 @@ function paperqsl_requested($qso_id, $method) {
 			);
 
 			$this->db->where('COL_PRIMARY_KEY', $qso_id);
+			$this->db->or_group_start();
 			$this->db->where('COL_QSL_SENT !=','R');
 			$this->db->where('COL_QSL_SENT_VIA !=', $method);
+			$this->db->group_end();
```

Comment:
To achieve that, what i assumed in my comment, try this instead:

```suggestion
            $this->db->group_start();
            $this->db->where('COL_QSL_SENT !=','R');
            $this->db->or_where('COL_QSL_SENT_VIA !=', $method);
            $this->db->group_end();
```
```

Reply:
Tnx!